recipes/_install.rb

```diff
@@ -37,6 +37,12 @@ def nsp
   notifies :delete, 'file[splunk-marker]', :immediately
 end
 
+service 'splunk-stop' do
+  service_name service
+  action :nothing
+  supports stop: true, nothing: true
```

Comment:
I don't think you need to list nothing here, that's not an actual action you would pass to the service.